EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,82 @@
+eip: xxxx  
```

Comment:
```suggestion
---
eip: 7825
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.